site/server/conf/evolutions/default/2.sql

```diff
@@ -0,0 +1,16 @@
+# --- !Ups
+
+DROP TABLE users;
+CREATE TABLE users (
+    id integer primary key,
+    login varchar(255) unique not null,
+    name varchar(255) not null,
+    github_id varchar(255) not null,
```

Comment:
Agreed with @raulraja 

Particularly, I also like table names in UpperCase and using non-plural entity names:

For instance:

`CREATE TABLE User (... , githubId varchar(255), ...)`

This naming convention prevents things like this in our persistence model:

`val user1 = Users(1, "", "" , .. )`

But this a personal preference. What do you think guys @raulraja  @dialelo ?